packages/base/spec.gts

```diff
@@ -136,10 +128,16 @@ export class Spec extends CardDef {
             {{/if}}
           </div>
           <div class='header-info-container'>
-            <h1 class='title' id='title' data-test-title><@fields.title /></h1>
-            <p class='description' data-test-description>
-              <@fields.description />
-            </p>
+            <div class='header-info-padding-container'>
+              <h1 class='title' id='title' data-test-title>
+                <@fields.title />
+              </h1>
+            </div>
+            <div class='header-info-padding-container'>
+              <p class='description' data-test-description>
+                <@fields.description />
+              </p>
+            </div>
```

Comment:
let's not do this template and css change here. we can create an edit template that handles the semantics and styles better. it can be outside of the scope of this PR.